src/lib/components/workflow/workflow-json-navigator.svelte

```diff
@@ -0,0 +1,114 @@
+<script lang="ts">
+  import CodeBlock from '$lib/holocene/code-block.svelte';
+  import RangeInput from '$lib/holocene/input/range-input.svelte';
+  import Loading from '$lib/holocene/loading.svelte';
+  import type { WorkflowEvents } from '$lib/types/events';
+
+  export let events: WorkflowEvents = [];
+  let index = 1;
+
+  function handleKeydown(event: KeyboardEvent) {
+    switch (event.code) {
+      case 'ArrowRight':
+      case 'KeyL':
+        event.preventDefault();
+        event.stopPropagation();
+        if (index < events.length) {
+          index += 1;
+        }
+        break;
+      case 'ArrowLeft':
+      case 'KeyH':
+        event.preventDefault();
+        event.stopPropagation();
+        if (index > 1) {
+          index -= 1;
+        }
+        break;
+      default:
+        break;
+    }
+  }
+</script>
+
+<svelte:window on:keydown={handleKeydown} />
+{#if !events.length}
+  <Loading />
+{:else}
+  <div class="flex gap-4">
+    <RangeInput
+      id="range-input-event-history"
+      min={1}
+      max={events.length}
+      bind:value={index}
+    />
+    <div class="flex items-center justify-center gap-3">
+      <button
+        class="caret"
+        disabled={index === 1}
+        on:click={() => {
+          index -= 1;
+        }}
+        aria-label="previous"
+      >
+        <span
+          class="arrow arrow-left"
+          class:arrow-left-disabled={index === 1}
+        />
+      </button>
+      <button
+        class="caret"
+        disabled={index === events.length}
+        on:click={() => {
+          index += 1;
+        }}
+        aria-label="next"
+      >
+        <span
+          class="arrow arrow-right"
+          class:arrow-right-disabled={index === events.length}
+        />
+      </button>
```

Comment:
Adding another `div` with breakpoints here makes it a bit easier to view on smaller screens 

<img width="374" alt="Screenshot 2023-05-12 at 11 21 19 AM" src="https://github.com/temporalio/ui/assets/15069288/68b09364-ec5c-408d-afec-2c4c6e421734">

```suggestion
  <div class="flex gap-4 max-sm:flex-col">
    <div class="flex w-full gap-4">
      <RangeInput
        id="range-input-event-history"
        min={1}
        max={events.length}
        bind:value={index}
      />
      <div class="flex items-center justify-center gap-3">
        <button
          class="caret"
          disabled={index === 1}
          on:click={() => {
            index -= 1;
          }}
          aria-label="previous"
        >
          <span
            class="arrow arrow-left"
            class:arrow-left-disabled={index === 1}
          />
        </button>
        <button
          class="caret"
          disabled={index === events.length}
          on:click={() => {
            index += 1;
          }}
          aria-label="next"
        >
          <span
            class="arrow arrow-right"
            class:arrow-right-disabled={index === events.length}
          />
        </button>
      </div>
```

Reply:
Updated in https://github.com/temporalio/ui/pull/1361